packages/core/lib/commands/create/helpers.js

```diff
@@ -70,81 +55,55 @@ function getLicense(options) {
   } catch {}
 }
 
-var Create = {
-  contract: function (directory, name, options, callback) {
-    if (typeof options === "function") {
-      callback = options;
-    }
-
-    var from = templates.contract.filename;
-    var to = path.join(directory, name + ".sol");
+const Create = {
+  contract: async function (directory, name, options) {
```

Comment:
Lately I've taken to doing:
```suggestion
  async contract (directory, name, options) {
```

(Have we talked about this recently?)

Could also make this an arrow function, since it doesn't look like there's a `this`